app/services/feeds/import.rb

```diff
@@ -24,26 +24,32 @@ def call
       users.in_batches(of: users_batch_size) do |batch_of_users|
         feeds_per_user_id = fetch_feeds(batch_of_users)
         Rails.logger.error("feeds::import::feeds_per_user_id.length: #{feeds_per_user_id.length}")
+        DatadogStatsClient.count("feeds::import::fetch_feeds.count", feeds_per_user_id.length)
 
         feedjira_objects = parse_feeds(feeds_per_user_id)
         Rails.logger.error("feeds::import::feedjira_objects.length: #{feedjira_objects.length}")
+        DatadogStatsClient.count("feeds::import::parse_feeds.count", feedjira_objects.length)
```

Comment:
Just a though, do you want to keep the name consistent with the Rails.logger?
```suggestion
        DatadogStatsClient.count("feeds::import::feedjira_objects.length", feedjira_objects.length)
```

Reply:
I actually wanted to change the logger, as "feedjra objects" doesn't mean much long term in the logs. I prefer counting the times the actions was invoked. I'll unify them but the other way :D